cmd/poller/collector/helpers.go

```diff
@@ -103,10 +102,26 @@ func (c *AbstractCollector) ImportSubTemplate(model, filename string, version [3
 	}
 
 	subTemplateFp = path.Join(pathPrefix, selectedVersion, filename)
-	c.Logger.Debug().Msgf("selected best-fitting subtemplate [%s]", subTemplateFp)
+	c.Logger.Info().Msgf("selected best-fitting subtemplate [%s]", subTemplateFp)
```

Comment:
intended?

Reply:
was for testing during draft PR. removed